Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -661,6 +667,23 @@ public void ScreenshotTransformationTest ()
       Helper.RunScreenshotTestExact<ScreenshotTest> (PrepareTest(), ScreenshotTestingType.Browser, test);
     }
 
+    private void RetryTest (Action action, int retries)
+    {
+      for (int i = 0; i < retries; i++)
+      {
+        try
+        {
+          action();
+        }
+        catch (AssertionException) when (i + 1 < retries)

```

Comment:
Hmm...is the condition wrong:
retries = 0: never continues because i=0, 0+1 < 0 == false
retries = 1: never continues because i=0, 0+1 < 1 == false
retries = 2: continues 1x because i=1, 1+1 < 2 == false

Reply:
Sorry the local

Reply:
Sorry I thought of the `retries` parameter as **tries** (total = `retries`) rather than **retries** (total = `1 + retries`) - this way, the condition makes sense to me, the last occurring `AssertionException` is not caught.

Which concept should I choose?
1. parameter indicates total tries
2. parameter indicates retries after an initial try

Reply:
"retries". otherwise, we could pass tries=0 and never do anything :p